src/Codeception/Module/DataFactory.php

```diff
@@ -173,6 +170,16 @@ public function _beforeSuite($settings = [])
             }
         }
     }
+    
+    /**
+     * @return StoreInterface
+     */
+    protected function getStore()
+    {
+        return $this->ormModule instanceof DataMappe
```

Comment:
Thanks for CR.
I will fix typo error, but tests...

I can\`t research and add functional tests now. 
Do we will can to add this PR without test? Now I don\`t find tests for DataFactory module - https://github.com/Codeception/Codeception/tree/2.2/tests/unit/Codeception/Module